ansible/README.md

```diff
@@ -149,6 +149,52 @@ You need to run `initdb` on cloudant **only once** per cloudant database to init
 **Hint:** The `initdb.yml` playbook will only initialize your database if it is not initialized already, else it will skip initialization steps.
 
 
+### Verification after Deployment
+
+After a successful deployment you can use the `wsk` CLI (located in the /bin folder of the repository) to verify that OpenWhisk is operable.
+
+1. Setup **authentication** for the CLI. This command uses the *guest* user, which is automatically added to your OpenWhisk deployment.
```

Comment:
I am inclined to remove this (see my comment above and Carlos' pull request with similar information).